builder/actions/git.py

```diff
@@ -32,15 +32,16 @@ def run(self, env):
 
         sh = env.shell
 
-        sh.exec("git", "clone", self.project.url, self.path, always=True)
+        sh.exec("git", "clone", self.project.url,
+                self.path, always=True, retries=3)
```

Comment:
retries. fucking neat

Reply:
I've seen every git command and apt command fail while testing. This is a necessity, I'm tired of "Well just push or run it again"

Reply:
I was serious.
Retries ARE neat.
It was cool to see you already had support for that